lib/sidekiq/client.rb

```diff
@@ -189,8 +189,23 @@ def enqueue_in(interval, klass, *args)
 
     def raw_push(payloads)
       @redis_pool.with do |conn|
-        conn.pipelined do |pipeline|
-          atomic_push(pipeline, payloads)
+        retryable = true
+        begin
+          conn.pipelined do
```

Comment:
You're missing some recent changes to `main`. With the latest redis-rb, this should be `conn.pipelined do |pipe|` and pass in the pipe.